<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentMain</class>
 <widget class="QDialog" name="studentMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>621</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="studentMainPage">
    <widget class="QPushButton" name="SMLogOutButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMWelcomeLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome </string>
     </property>
    </widget>
    <widget class="QLabel" name="SMNameLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMUpcomingAssignmentsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Upcoming Assignments:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="SMUpcomingAssignmentsList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>331</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="SMUpcomingTestsList">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>380</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="SMUpcomingTestsLabel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>350</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Upcoming Tests:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMCompletedTasksLabel">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Completed Assignments/Tests:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="SMCompletedTasksList">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SMStartSelectedAssignmentButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>520</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Seleced Assignment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMStartSelectedTestButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>520</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Selected Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMViewAccountDataButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>View Account Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMViewGradesButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>View Grades</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SMAccountDetailsPage">
    <widget class="QPushButton" name="SMChangePasswordButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMChangeEmailButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMChangeSurnameButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>160</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Surname</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMChangeNameButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMAccountDetailsBackButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADIDLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentID">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADSurnameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentSurname">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADDateOfBirthLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date of Birth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentDateOfBirth">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADEmailLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentEmail">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCourceCodeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cource Code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentCourceCode">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMADPasswordLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMADCurrentPassword">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>390</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SMSolveTheTestPage">
    <widget class="QRadioButton" name="SMSTTAnswer1Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>471</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="SMSTTAnswer2Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="SMSTTAnswer3Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>390</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="SMSTTAnswer4Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>440</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SMSTTNextQuestionButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>510</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Question</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTTQuestionLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>701</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTTCorrectAnswersLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTTSlashLabel">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTTAllAnswersLabel">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PTS.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SMSendTheAssignmentPage">
    <widget class="QLabel" name="SMSTALabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Send Assignment</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTATitleLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="SMSTADescriptionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="SMSTAAddFileButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMSTASendAssignmentButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>510</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Assignment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SMSTACancelButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QListWidget" name="SMSTAFileList">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SMViewGradesPage">
    <widget class="QTableWidget" name="SMVGTable">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>661</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SMVGBackButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>550</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
